src/main/java/pack/util/ResponseFormat.java

```diff
@@ -0,0 +1,41 @@
+package pack.util;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+@Component
+public class ResponseFormat {
+	final static JsonNodeFactory factory = JsonNodeFactory.instance;
```

Comment:
After you use ResponseEntity you can completely remove ResponseFormat.

Reply:
deleted